com.microsoft.Edge.yaml

```diff
@@ -0,0 +1,155 @@
+app-id: com.microsoft.Edge
+# desktop-file-name-prefix: '(Testing) '
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: edge
+separate-locales: false
+build-options:
+  no-debuginfo: true
+tags:
+  - proprietary
+finish-args:
+  - '--device=all'
+  - '--env=GTK_PATH=/app/lib/gtkmodules'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=cups'
+  - '--socket=pulseaudio'
+  - '--socket=x11'
+  - '--socket=wayland'
+  - '--require-version=1.8.2'
+  - '--system-talk-name=org.freedesktop.UPower'
+  - '--talk-name=org.freedesktop.FileManager1'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.freedesktop.secrets'
+  - '--talk-name=org.gnome.SessionManager'
+  - '--system-talk-name=org.freedesktop.Avahi'
+  - '--filesystem=xdg-run/pipewire-0'
+  - '--own-name=org.mpris.MediaPlayer2.chromium.*'
+#  - '--filesystem=xdg-documents'
+  - '--filesystem=xdg-download'
+#  - '--filesystem=xdg-music'
+#  - '--filesystem=xdg-videos'
+modules:
+  - shared-modules/libsecret/libsecret.json
+
+  - name: libcups
+    make-args: [libs]
+    no-make-install: true
+    post-install:
+      - make install-headers install-libs
+    cleanup:
+      - /include
+    sources:
+      - type: archive
+        url: https://github.com/apple/cups/releases/download/v2.3.3/cups-2.3.3-source.tar.gz
+        sha256: 261fd948bce8647b6d5cb2a1784f0c24cc52b5c4e827b71d726020bcc502f3ee
+
+  - name: gtk-cups-backend
+    buildsystem: meson
+    make-args: [modules/printbackends/libprintbackend-cups.so]
+    no-make-install: true
+    post-install:
+      - install -Dm 755 modules/printbackends/libprintbackend-cups.so -t /app/lib/gtkmodules/3.0.0/printbackends/
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/gtk.git
+        tag: 3.24.23
+
+  - name: gtk-settings
+    buildsystem: simple
+    build-commands:
+      - 'install -Dm 644 gtk-settings.ini /app/etc/xdg/gtk-3.0/settings.ini'
+    sources:
+      - type: file
+        path: gtk-settings.ini
+
+  - name: pipewire
```

Comment:
pipewire 0.3 is part of the runtime. Why this older version?

Reply:
I only used this older version because [Chrome](https://github.com/flathub/com.google.Chrome/blob/beta/com.google.Chrome.yaml#L78) and [Brave](https://github.com/flathub/com.brave.Browser/blob/beta/com.brave.Browser.yaml#L75) use the same version.

Reply:
Upstream Chromium currently only supports pipewire 0.2 (the ABI changed in 0.3), I believe there are either open or already merged change requests upstream to support 0.3 but it won't trickle down for a few releases.

Reply:
fair enough